deployments/automated-previews.mdx

```diff
@@ -207,3 +207,51 @@ destroy_preview:
     - if: $CI_MERGE_REQUEST_ID
       when: manual
 ```
+
+## BitBucket Pipelines
+
+### Creating and cleaning up preview environments
+
+This job can be pasted into your `bitbucket-pipelines.yml` at the root of your repository. Be sure to add repository variables for the ones mentioned in the warning below.
+
+On pull request creation, an Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and
+deployed to the environment. The pipeline provides the option to remove the deployed component and its environment from the Architect Cloud manually,
+which should be done once the pull request is merged or closed.
+
+<Warning>
+  This example assumes that the repo has `ARCHITECT_EMAIL`, `ARCHITECT_PASSWORD`, `ARCHITECT_ACCOUNT`, and `ARCHITECT_CLUSTER` set as CI/CD variables.
+</Warning>
+
+<Warning>
+  `ARCHITECT_PASSWORD` must be a [personal access
+  token](https://cloud.architect.io/login).
+</Warning>
+
+```yml
+image: docker:23.0.1
+
+pipelines:
+  pull-requests:
+    '**':
+      - step:
+          name: 'Deploy preview'
+          services:
+            - docker
+          script:
+            - apk add --update npm
+            - npm install -g @architect-io/cli
+            - architect login
+            - architect environment:create preview-$BITBUCKET_PR_ID --description="https://bitbucket.com/$BITBUCKET_WORKSPACE/$BITBUCKET_REPO_SLUG/pull-requests/$BITBUCKET_PR_ID"
+            - architect deploy ./architect.yml --environment preview-$BITBUCKET_PR_ID --auto-approve
+      - step:
+          name: 'Destroy preview'
+          trigger: 'manual'
```

Comment:
so we make this a manual trigger? can't we kick this off on PR close?

Reply:
Unfortunately, BitBucket's CI/CD pipelines aren't that robust and there isn't a clean way to do that. Details from this link in the comment above - https://jira.atlassian.com/browse/BCLOUD-13867